draft-regext-simple-contact.md

```diff
@@ -332,14 +254,130 @@ of strings. Each string MUST be conformant to the geo URI scheme as defined in [
 
 "geo" values SHOULD NOT be used with contact data when "kind" is "individual".
 
+## An Entity Example {#example}
+
+The following is an example of an RDAP entity using SimpleContact:
+
+    {
+      "objectClassName" : "entity",
+      "handle":"XXXX",
+      "sc_data": {
+        "kind" : "individual",
+        "individualNames" : [
+          {
+            "name" : "Dr. Joe User Jr., Ph.d.",
+            "lang" : "en-US"
+          }
+        ],
+        "roleNames" : [
+          {
+            "name" : "Registration Services Help Desk",
+            "lang" : "en-US"
+          }
+        ],
+        "organizationNames" : [
+          {
+            "name" : "ACME Pty",
+            "lang" : "en-US"
+          }
+        ],
+        "postalAddresses" : [
+          {
+            "completeAddress" : [
+              "Suite 300",
+              "123 Random Tree Name Street",
+              "Kalamazoo, MI 90125 US"
+            ]
+            "regionCode" : "MI",
+            "countryCode" : "US",
+            "postalCode" : "90215",
+            "lang" : "en-US"
+          }
+        ],
+        "emails" : [
+          {
+            "email" : "joe@example.net"
+          }
+        ],
+        "voicePhones" : [
+          {
+            "phone" : "+447040202"
+          }
+        ],
+        "faxPhones" : [
+          {
+            "phone" : "tel:+1-201-555-9999;ext=123",
+            "masked" : false
+          }
+        ],
+        "webContacts" : [
+          {
+            "uri" : "https://example.net/contact-me"
+          }
+        ]
+      },
+      "roles":[ "registrar" ],
+      "publicIds":[
+        {
+          "type":"IANA Registrar ID",
+          "identifier":"1"
+        }
+      ],
+      "remarks":[
+        {
+          "description":[
+            "She sells sea shells down by the sea shore.",
+            "Originally written by Terry Sullivan."
+          ]
+        }
+      ],
+      "links":[
+        {
+          "value":"https://example.com/entity/XXXX",
+          "rel":"self",
+          "href":"https://example.com/entity/XXXX",
+          "type" : "application/rdap+json"
+        },
+        {
+          "value":"https://example.com/entity/XXXX",
+          "rel":"about",
+          "href":"https://example.com/entity/XXXX.vcard",
+          "type" : "text/vcard"
+        }
+      ],
+      "events":[
+        {
+          "eventAction":"registration",
+          "eventDate":"1990-12-31T23:59:59Z"
+        }
+      ],
+      "asEventActor":[
+        {
+          "eventAction":"last changed",
+          "eventDate":"1991-12-31T23:59:59Z"
+        }
+      ]
+    }
```

Comment:
Perhaps, to help stress-test, good to also give an example for a lesser known economy, unlike the US.

Reply:
I asked Bard for "weird" addresses and Galapagos Islands, Finland (not a lesser-known economy but with quite a different language), and Antartica came up. We could just settle for say Japan given we use such an example for email. This is simply a suggestion to give a non-US example but your call in the end.